content/collections/reports/.gitignore

```diff
@@ -2,4 +2,4 @@
 *
 # Except this file and labeldefault.json
 !.gitignore
-!labeldefault.json
\ No newline at end of file
+!label.json
```

Comment:
Given that this file *could* and is kinda meant to vary across portals and just acts to seed the row upon first use, I don't think it should be tracked by git. Thoughts? Is the labeldefault.json addition here relatively new, @egbot ?

Reply:
Incidentally, the only mention of labeldefault.json in the entire codebase seems to be in content/collections/reports/.gitignore.... not sure whether it's cruft that needs to be removed or whether the implementation of it has been broken.

Reply:
I think labeldfault.json is not supposed to be called by the code. It is a file that exists as a default/template so that someone can create a label.json based off of it.

Reply:
Oh of course. That makes total sense. In which case, vote we keep tracking the default but not label.json.

Reply:
It's a good question.  I was setting it to be tracked as a convenient way to distribute  it to existing portals and make sure that the value gets pulled into the DB.  Worth noodling on for a bit.

Reply:
I updated the update doc linked in the pull request checklist, but nothing in the readme. I'll code review asap.